_episodes/16-parallel.md

```diff
@@ -111,6 +113,18 @@ If we run the Python script locally with a command-line parameter, as in
 ```
 {: .language-bash}
 
+> ## Random Number Generation
```

Comment:
Erroneous duplication of the RNG discussion box

Reply:
This has been removed.